### DISCUSSION TITLE
Not identical visitor_id

### DISCUSSION THREAD

**Discussion by s-fog** [fingerprintjs/fingerprintjs#626] [General]
Hello. Need some help.
We called this fingerprint in two different places. Here http://salyakhov.kz/demo.html and here https://dmp.one/fpjs 
And we get different fingerprints. Can u tell where is problem? 

https://dmp.one/fpjs here fingerprint is directly on the page

Here http://salyakhov.kz/demo.html u can see request to http://176.110.125.49:8000/dmp

**Comment by Finesse** [comment]
Fingerprint are different on HTTP and HTTPS pages because some browser APIs are unavailable under HTTP. You should use the same protocol on both pages. You can see which fingerprint components are different by comparing the contents of `result.components`.

**Reply by s-fog** [reply]
Thanks for help!

**Reply by Valve** [reply]
@s-fog please mark as resolved if the answer above helped you
